components/org.wso2.carbon.identity.organization.management.service/src/main/java/org/wso2/carbon/identity/organization/management/service/util/OrganizationManagementConfigBuilder.java

```diff
@@ -177,4 +204,70 @@ private String replaceSystemProperty(String text) {
         }
         return textBuilder.toString();
     }
+
+    private void buildCacheConfig() {
+
+        OMElement cacheConfig = this.documentElement.getFirstChildWithName(new QName(CACHE_CONFIG));
```

Comment:
We could do "return;" after checking if cases in the reverse logic and stop this code growing to the right.